deploy/rbac-secretproviderclass.yaml

```diff
@@ -32,6 +32,14 @@ rules:
   - get
   - list
   - watch
+- apiGroups:
```

Comment:
Please revert changes to `deploy` folder. All the yaml changes in `manifest_staging` will be promoted into the released charts folder during the release.